include/mbedtls/mbedtls_config.h

```diff
@@ -2534,16 +2534,40 @@
 #define MBEDTLS_ECJPAKE_C
 
 /**
- * \def MBEDTLS_ECP_C
+ * \def MBEDTLS_ECP_LIGHT
  *
  * Enable the elliptic curve over GF(p) library.
+ * This does not include support for scalar multiplication and functionalities
+ * that depepend on it, which include:
+ * - interruptible/restartable operations
+ * - reading of compressed EC points
+ * - parsing of explicit (non-named) curves
+ * - EC key derivation in PSA.
+ *
+ * \note This list of limitations is expected to change in the future as soon
+ *       as new functionalities get implemented (ex: through PSA's drivers).
+ * \note Enabling MBEDTLS_PK_PARSE_EC_EXTENDED cause MBEDTLS_ECP_LIGHT to be
+ *       automatically enabled for backward compatibilty reasons.
+ *
+ * Module:  library/ecp.c
+ *
+ * Requires: MBEDTLS_BIGNUM_C and at least one MBEDTLS_ECP_DP_XXX_ENABLED
+ */
+#define MBEDTLS_ECP_LIGHT
+
+/**
+ * \def MBEDTLS_ECP_C
+ *
+ * This add support for scalar multiplication in ECP module thus overcoming
```

Comment:
While this is correct, I think it would be clearer to say "Enable all of the elliptic curve over GF(p) library."